### PR TITLE
[INFRA] add citation cff and validation workflow

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to #76 

Changes proposed in this pull request:

- add a citation file and a workflow to validate it

## Checklist

- [ ] PR has an interpretable title with a prefix (`[ENH]`, `[BUG]`, `[DOC]`, `[INFRA]`, `[MAINT]`)
- [ ] PR links to Github issue with mention `Closes #XXXX`
- [ ] Tests pass
- [ ] Code is properly formatted

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  - affiliation: "ORIGAMI lab, McGill University, Montréal, Canada"
+    family-names: Jahanpour
+    given-names: Arman    
+
+  - affiliation: "ORIGAMI lab, McGill University, Montréal, Canada"
+    email:  nikhil.bhagwat@mcgill.ca
+    family-names: Bhagwat
+    given-names: Nikhil
+    orcid: "https://orcid.org/0000-0001-6073-7141"
```
Comment (Remi-Gau) [inline CITATION.cff:39]: @jarmoza @jarmoza @alyssadai @surchs 
you can add your ORCID if you want

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,46 @@
+cff-version: 1.2.0
+
+title: "neurobagel_api"
+
+version: 0.2.0
```
[the remark this answers is not in the document]
Comment (surchs) [reply CITATION.cff:8]: @alyssadai if you want to release as 0.1.0, please adjust this line accordingly

File: .github/workflows/cff_validation.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: validate citation.cff
+
+on:
+  push:
+    branches: ['*']
```
Comment (surchs) [inline .github/workflows/cff_validation.yml:6]: I think we can probably just run this against `main` since we know what our branches are called.

**Review by surchs — APPROVED** [review]
Thanks a lot @Remi-Gau for getting us started on this. This is good to go :cook: I'll merge it for you!

**Pull request merged**